frontend/src/api/services/answer_survey.test.ts

```diff
@@ -77,4 +80,52 @@ describe('AnswerSurveyService', () => {
       });
     });
   });
+
+  describe('get method', () => {
+    describe('when returns expected data', () => {
+      beforeEach(async () => {
+        // mocking post implementation to return the expected value
+        (httpClient as jest.Mocked<AxiosInstance>).post.mockImplementationOnce(
+          jest.fn((url: string, body: AnswerSurveyGet) => {
+            return Promise.resolve({ data: mockResponse });
+          })
+        );
+
+        instance = new AnswerSurveyService(httpClient as any);
+
+        res = await instance.get(mockAnswerSurveyGet);
+      });
+
+      it('should call get method once', () => {
+        expect(httpClient.get).toHaveBeenCalled();
+      });
+
+      it('should return expected data', () => {
```

Comment:
is expected data `expect(res).toEqual(undefined);`?

Reply:
Yep!

` public async get(
    answer_survey: AnswerSurveyGet
  ): Promise<AxiosResponse<AnswerSurveyReceive | undefined>> {
    try {
      const response = await this.httpClient.get<AnswerSurveyReceive>(
        `api/v1/answers_surveys/${answer_survey.id}`
      );

      return response;
    } catch (error) {
      errorHandler(error);
    }

    return undefined;
  }`

The service can return undefined if try not called.